src/language/grammar.tx

```diff
@@ -4,18 +4,27 @@ Specification:
 	constraints += Constraint*
 ;
 
-Param: key = ID ':' value = STRING ;
+Param: 'Param' key = ID ':' value = STRING ;
 
 Actor: Car | Pedestrian ;
 
 Car:
 	'Car' name = ID
+	isEgo ?= 'isEgo'
+	( 'color' ':' color = ID ) ?
+	( 'speed' ':' speed = STRING ) ?
```

Comment:
What kind of things can we possibly write for speed? For now I set the type to `STRING` instead of `ID`, because `ID` has to be of a format like variable names (alphanumerical, starts with letter, etc.), but now with `STRING` we would have to put quotation marks

Reply:
Resolved. Changed it to a custom `SPEED` type with `FLOAT | ID`